### ISSUE TITLE
MSW definitions are non-deterministic

### ISSUE THREAD

**Issue by arlyon** [anymaniax/orval#703]
## What are the steps to reproduce this issue?

1. Run a build with MSW outputs
3. Run a build again

## What happens?

Observe that some msw objects' keys tend to swap places at random causing noisy diffs.

## What were you expecting to happen?

I expect the keys to be sorted, or to match the order from the openapi spec.

## What versions are you using?

**Operating System:** fedora
**Package Version:** latest
**Browser Version:** irrelevant

**Comment by blentz100** [comment]
I'm noticing the same behavior with model and controller output. The order of things seems to be different for each run of Orval resulting in noisy diffs. Would love to hear if anyone has a solution for this. Thanks.

**Comment by arlyon** [comment]
No, we're just dealing w/ the noise for now. I am considering taking the time to write a PR over christmas.

**Comment by anymaniax** [comment]
Sorry guys I don't have the time to check this right now. I will probably have the time early january

**Comment by blentz100** [comment]
@anymaniax Thank you. I appreciate the library and the work you are doing on this. Enjoy the holidays.

**Comment by anymaniax** [comment]
@blentz100 thanks and enjoy the holidays too!
